### DISCUSSION TITLE
Welcome to sdwh.github.io Discussions!

### DISCUSSION THREAD

**Discussion by sdwh** [sdwh/sdwh.github.io#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by peter0083** [comment]
剛剛下載了EZ100PU讀卡機驅動程式, 感謝大大分享
